server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewPeriodServicesImpl.java

```diff
@@ -21,14 +32,34 @@ class ReviewPeriodServicesImpl implements ReviewPeriodServices {
     private static final Logger LOG = LoggerFactory.getLogger(ReviewPeriodServicesImpl.class);
 
     private final ReviewPeriodRepository reviewPeriodRepository;
+    private final ReviewAssignmentRepository reviewAssignmentRepository;
+    private final MemberProfileRepository memberProfileRepository;
     private final ReviewStatusTransitionValidator reviewStatusTransitionValidator;
-
-    ReviewPeriodServicesImpl(
-            ReviewPeriodRepository reviewPeriodRepository,
-            ReviewStatusTransitionValidator reviewStatusTransitionValidator
-    ) {
+    private EmailSender emailSender;
+    private final Environment environment;
+    private final String webAddress;
+    public static final String WEB_ADDRESS = "check-ins.web-address";
+
+    @Value("${aes.key}")
+    private String aesKey;
+
+    public ReviewPeriodServicesImpl(ReviewPeriodRepository reviewPeriodRepository,
+                                    ReviewAssignmentRepository reviewAssignmentRepository,
+                                    MemberProfileRepository memberProfileRepository,
+                                    CurrentUserServices currentUserServices,
+                                    @Named(MailJetConfig.HTML_FORMAT) EmailSender emailSender,
+                                    Environment environment, @Property(name = WEB_ADDRESS) String webAddress) {
         this.reviewPeriodRepository = reviewPeriodRepository;
+        this.reviewAssignmentRepository = reviewAssignmentRepository;
+        this.memberProfileRepository = memberProfileRepository;
         this.reviewStatusTransitionValidator = reviewStatusTransitionValidator;
+        this.emailSender = emailSender;
+        this.environment = environment;
+        this.webAddress = webAddress;
+    }
```

Comment:
This didn't compile on my machine.. shouldn't it be this...?  Maybe the merge messed it up?

```suggestion
    public ReviewPeriodServicesImpl(ReviewPeriodRepository reviewPeriodRepository,
                                    ReviewAssignmentRepository reviewAssignmentRepository,
                                    MemberProfileRepository memberProfileRepository,
                                    ReviewStatusTransitionValidator reviewStatusTransitionValidator,
                                    @Named(MailJetConfig.HTML_FORMAT) EmailSender emailSender,
                                    Environment environment,
                                    @Property(name = WEB_ADDRESS) String webAddress) {
        this.reviewPeriodRepository = reviewPeriodRepository;
        this.reviewAssignmentRepository = reviewAssignmentRepository;
        this.memberProfileRepository = memberProfileRepository;
        this.reviewStatusTransitionValidator = reviewStatusTransitionValidator;
        this.emailSender = emailSender;
        this.environment = environment;
        this.webAddress = webAddress;
    }
```

Reply:
Oh I see what I did wrong. I have an unpushed change that fixes the above. Thank you very much for the suggestions and pointing out the branch naming issue! I will include the suggestions then create a new branch/PR.